dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -532,3 +538,227 @@ def min_time_limit(self, dqm):
+
+        if FeatureFlags.hss_solver_config_override:
+            # use legacy behavior (override solver config from env/file)
+            solver = config.setdefault('solver', {})
+            if isinstance(solver, abc.Mapping):
+                solver.update(self.default_solver)
+
+        # prefer the latest hybrid CQM solver available, but allow for an easy
+        # override on any config level above the defaults (file/env/kwarg)
+        defaults = config.setdefault('defaults', {})
+        if not isinstance(defaults, abc.Mapping):
+            raise TypeError("mapping expected for 'defaults'")
+        defaults.update(solver=self.default_solver)
+
+        self.client = Client.from_config(**config)
+        self.solver = self.client.get_solver()
+
+        # For explicitly named solvers:
+        if self.properties.get('category') != 'hybrid':
+            raise ValueError("selected solver is not a hybrid solver.")
+        if 'cqm' not in self.solver.supported_problem_types:
+            raise ValueError("selected solver does not support the 'cqm' problem type.")
+
+    @classproperty
+    def default_solver(cls) -> Dict[str, str]:
+        """Features used to select the latest accessible hybrid CQM solver."""
+        return dict(supported_problem_types__contains='cqm',
+                    order_by='-properties.version')
+
+    @property
+    def properties(self) -> Dict[str, Any]:
+        """Solver properties as returned by a SAPI query.
+
+        `Solver properties <https://docs.dwavesys.com/docs/latest/c_solver_properties.html>`_
+        are dependent on the selected solver and subject to change.
+        """
+        try:
+            return self._properties
+        except AttributeError:
+            self._properties = properties = self.solver.properties.copy()
+            return properties
+
+    @property
+    def parameters(self) -> Dict[str, List[str]]:
+        """Solver parameters in the form of a dict, where keys
+        are keyword parameters accepted by a SAPI query and values are lists of
+        properties in
+        :attr:`~dwave.system.samplers.LeapHybridCQMSampler.properties` for each
+        key.
+
+        `Solver parameters <https://docs.dwavesys.com/docs/latest/c_solver_parameters.html>`_
+        are dependent on the selected solver and subject to change.
+        """
+        try:
+            return self._parameters
+        except AttributeError:
+            parameters = {param: ['parameters']
+                          for param in self.properties['parameters']}
+            parameters.update(label=[])
+            self._parameters = parameters
+            return parameters
+
+    def sample_cqm(self, cqm: dimod.ConstrainedQuadraticModel,
+                   time_limit: Optional[float] = None, **kwargs):
+        """Sample from the specified constrained quadratic model.
+
+        Args:
+            cqm (:obj:`dimod.ConstrainedQuadraticModel`):
```

Comment:
`:class:`

Reply:
Blame @JoelPasvolsky :laughing:, I'm also going to leave it alone to keep the CI time reasonable.